django/db/models/functions/datetime.py

```diff
@@ -264,7 +264,7 @@ def convert_value(self, value, expression, connection):
 
 class Trunc(TruncBase):
 
-    def __init__(self, expression, kind, output_field=None, tzinfo=None, is_dst=None, **extra):
+    def __init__(self, expression, kind, output_field=None, tzinfo=None, is_dst=timezone.NOT_PASSED, **extra):
```

Comment:
We can miss removing `is_dst` when bootstrapping Django 5.0, maybe a small comment.
```suggestion
    # RemovedInDjango50Warning: when the deprecation ends, remove is_dst
    # argument.
    def __init__(self, expression, kind, output_field=None, tzinfo=None, is_dst=timezone.NOT_PASSED, **extra):
```